rasa/core/evaluation/markers.py

```diff
@@ -56,11 +56,11 @@ def from_file(cls, path: Text) -> Dict:
     def from_yaml(cls, yaml: Text, filename: Text = "") -> Dict:
         """Loads the config from YAML text after validating it."""
         try:
-            # TODO implement and call the yaml schema validation
-            #  rasa.shared.utils.validation.validate_yaml_schema(yaml,
-            #  MAKERS_SCHEMA_FILE)
 
-            return rasa.shared.utils.io.read_yaml(yaml)
+            config = rasa.shared.utils.io.read_yaml(yaml)
+            cls._validate_config(config)
```

Comment:
Shouldn't `filename` argument be provided here too?

Reply:
Well spotted.